sdk/include/psdk/dwmapi.h

```diff
@@ -177,6 +230,12 @@ typedef struct _DWM_THUMBNAIL_PROPERTIES
     BOOL  fSourceClientAreaOnly;
 } DWM_THUMBNAIL_PROPERTIES, *PDWM_THUMBNAIL_PROPERTIES;
 
+#define DWM_TNP_RECTDESITIONATION     0x00000001
```

Comment:
Not sure why you're adding these yet again (and notice that this define name has a typo...)
And these defines are already defined [just above](https://github.com/reactos/reactos/pull/7902/files#diff-ffd023738fa428654929dcc457b065c6d75686613c94af70fcf38d765e1e4df7R217).

Side-note: Remember you asked yesterday on Discord what the signification of these defines could be? It's explained [here in MSDN](https://learn.microsoft.com/en-us/windows/win32/dwm/dwm-tnp-constants).